eslint.config.mjs

```diff
@@ -12,8 +12,7 @@ export default [
             parser: typescript,
             parserOptions: {
                 ecmaVersion: "latest",
-                sourceType: "module",
-                project: "./tsconfig.json", // Make sure your tsconfig includes @types/node
```

Comment:
how does eslint know where to look for the typescript config now?

Reply:
fixed